components/views/login/Login.module.scss

```diff
@@ -0,0 +1,81 @@
+.content {
+    display: grid;
+    place-items: center;
+    height: calc(100vh - var(--nav-height) - var(--footer-height));
+    .formContainer {
+        height: fit-content;
+        width: 20rem;
+        background: var(--darkClearBg);
+        padding: 2rem;
+        display: flex;
+        flex-direction: column;
+        .header {
+            text-align: center;
+            color: var(--off-white);
+            padding-bottom: 1rem;
+        }
+        .errorMessage {
+            color: var(--error-red);
```

Comment:
Drilling down a bit more here, instead of `--error-red`, we should do `error-font`. Cuz we dont care that the color is red, the variable name doesnt have to reflect the color. The thinking behind this is we have a set of theme variables related to errors.

```
--error-font: {whatever red you chose}
--error-background: {a complimenting, lighter color red to put as the background when we need it}
```
No need to add the error-background now. Just laying the ground work and theory behind why and how to name variables